CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 0.8.8 (TBD, 2018)
+## 0.8.9 (TBD 2018)
```

Comment:
Recommend putting this as (August TBD, 2018) since we officially abandon support for Python 2.7 on August 31, 2018.

Also, since this is prepping for the 0.8.9 release for Python 2.7, you should bump the version number to 0.8.9 in this PR in all 4 required places:
* setup.py
* cmd2.py
* docs/conf.py
* tests/test_cmd2.py

NOTE: On *master* now, we have completely changed the version scheme so there is no explicit version number anymore (it is based on the Git Tag).